omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/clinicianfacing/VisitsSectionFragmentController.java

```diff
@@ -38,14 +42,14 @@
  * Supports the containing PageModel having an "app" property whose config defines a "visitUrl" property
  */
 public class VisitsSectionFragmentController {
-
+	protected final Log log = LogFactory.getLog(this.getClass());

```

Comment:
I will do just that. 
And as for the issue @djazayeri  raised, I addressed it in my comment above.